### ISSUE TITLE
Background, Blackout Screen, and CloseOnClickAway Do Not Work on Mobile

### ISSUE THREAD

**Issue by anose001** [AvaloniaUtils/DialogHost.Avalonia#66]
The left side on the picture below shows the Dialog as it should be displayed with a white background and the blackout screen overlayed over the rest of the screen on desktop. The right side shows the mobile case where neither the background nor the overlayed blackout screen display. `CloseOnClickAway="True" ` does not work on mobile but works on desktop. The logic connected to the viewmodels works properly. 

![Issue3](https://github.com/user-attachments/assets/eff1b6e6-76da-4378-a942-b8584d3dd624)

**Comment by anose001** [comment]
On desktop `Background` and `OverlayBackground `properties have default values White and Black. This is not the case for mobile. Setting `Background="White"` and `OverlayBackground="Black"` explicitly fixes the above issues, including the `CloseOnClickAway` as the background is now clickable.

**Comment by SKProCH** [comment]
Its actually doesn't have White and Black values, it searches for this resources:
```xaml
    <Setter Property="OverlayBackground"
            Value="{utilities:MultiDynamicResource DefaultBrush=Black, ResourceKeys=SystemControlBackgroundBaseHighBrush}" />
    <Setter Property="Background"
            Value="{utilities:MultiDynamicResource DefaultBrush=Black, ResourceKeys=SystemControlBackgroundAltHighBrush;MaterialDesignPaper;MaterialPaperBrush}">
```

Probably by some conditions it will doesn't find it on mobile. Does this reproduceable in blank project with Fluent theme only?

**Comment by anose001** [comment]
I have not been using th FluentTheme by default, maybe this is where this issue stems from. I would have to set it up first and then test it

**Comment by SKProCH** [comment]
If it reproduceable with Fluent theme its a valid bug and should be fixed.

**Comment by anose001** [comment]
@SKProCH , I added the Fluent theme package and added the style to my app settings and the background and blackout screen do not behave as expected. I guess the above is a quick workaround for this.

**Comment by SKProCH** [comment]
Should be fixed with `v0.9.0-preview1`. If no - please, reopen the issue.
